nuxeo-i18n-behavior.html

```diff
@@ -54,7 +54,7 @@
       },
 
       refreshI18n() {
-        this.i18n = null;
+        // this.i18n = null;
```

Comment:
I'm assuming this was here to force the refresh because otherwise, since the `translate` function doesn't actually change, it won't trigger any refresh, no?

Reply:
Ha! Now I see I'm the one that commented out that line in master 😅